Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue

```diff
@@ -17,150 +16,139 @@ interface TourSlide {
-            imageUri: new URL(
-                "@/assets/images/tour/mobile/at-quick-link-mobile.gif",
-                import.meta.url
-            ).href,
-            imageAlt: "Quick link mobile demo",
-        },
-        {
-            title: "Filter your Timeline records",
-            description:
-                "Filter by health record type, date or keyword to find what you need.",
-            imageUri: new URL(
-                "@/assets/images/tour/mobile/at-filter-mobile.gif",
-                import.meta.url
-            ).href,
-            imageAlt: "Timeline filter mobile demo",
-        },
-        {
-            title: "Notifications centre",
-            description:
-                "You'll be notified of updates to the app and your health records.",
-            imageUri: new URL(
-                "@/assets/images/tour/mobile/at-notifications-mobile.gif",
-                import.meta.url
-            ).href,
-            imageAlt: "Notifications centre mobile demo",
-        },
-    ];
+const slideIndex = ref(0);
+const isVisible = ref(false);
 
-    private desktopSlides: TourSlide[] = [
-        {
-            title: "App Tour",
-            description:
-                "Take this brief tour to learn how to use Health Gateway. You can always get here again by clicking the light bulb.",
-            imageUri: new URL(
-                "@/assets/images/tour/web/at-intro.png",
-                import.meta.url
-            ).href,
-            imageAlt: "App tour start splash",
-        },
-        {
-            title: "Add a Quick Link",
-            description:
-                "Add a quick link to easily access a health record type from your home screen.",
-            imageUri: new URL(
-                "@/assets/images/tour/web/at-quick-link.png",
-                import.meta.url
-            ).href,
-            imageAlt: "Quick link web demo",
-        },
-        {
-            title: "Filter your Timeline records",
-            description:
-                "Filter by health record type, date or keyword to find what you need.",
-            imageUri: new URL(
-                "@/assets/images/tour/web/at-filter.png",
-                import.meta.url
-            ).href,
-            imageAlt: "Timeline filter web demo",
-        },
-        {
-            title: "Notifications centre",
-            description:
-                "You'll be notified of updates to the app and your health records.",
-            imageUri: new URL(
-                "@/assets/images/tour/web/at-notifications-web.png",
-                import.meta.url
-            ).href,
-            imageAlt: "Notifications centre web demo",
-        },
-    ];
+const tourCarousel = ref<BCarousel>();
 
-    private slideIndex = 0;
+const isMobile = computed<boolean>(() => store.getters["isMobile"]);
 
-    private isVisible = false;
+const slides = computed<TourSlide[]>(() => {
+    return isMobile.value ? mobileSlides : desktopSlides;
+});
```

Comment:
Is it okay to re-enable this behaviour or should we return desktopSlides in both cases for now?

Reply:
Yes you're right only desktopSlides still.